modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionFrontendTokenDefinition/testShouldCreateConfigFileWithFrontendTokenDefinitionJson.gradle

```diff
@@ -0,0 +1,25 @@
+import groovy.json.JsonSlurper
+
+task runGradleTest
+
+runGradleTest {
+	dependsOn ":client-extension:any-theme-css-project-create-config-file-frontend-token-definition:build"
+	doLast {
+		Project project = project(":client-extension:any-theme-css-project-create-config-file-frontend-token-definition")
+
+		File buildDir = new File(project.buildDir, "liferay-client-extension-build")
+
+		File configJSONFile = new File(buildDir, "any-theme-css-project-create-config-file-frontend-token-definition.client-extension-config.json")
+
+		assert configJSONFile.exists()
+
+		def jsonData = new JsonSlurper().parse(configJSONFile)
+
+		jsonData.each {
+			key, value ->
+			assert value instanceof Map
+			assert value.get("typeSettings") == ["frontendTokenDefinitionJSON={\"test\":123}"]: "'frontendTokenDefinitionJSON' key should exists with correct json content. Actual: ${value.get("typeSettings")}"
```

Comment:
These are two statements and the order matters, there should be an empty line between them